test/index.js

```diff
@@ -152,6 +152,14 @@ describe('Public API', () => {
             expect(returnedValue).to.equal(api);
         });
 
+        it('should be allowed to use without addEntry() or addStyleEntry()', () => {
+            api.setOutputPath('/tmp')
+                .setPublicPath('/tmp')
+                .copyFiles({ from: './' });
+            const webpackConfig = api.getWebpackConfig();
+            expect(Object.keys(webpackConfig.entry).length).to.equal(1);
+        });
```

Comment:
This test should probably be in https://github.com/symfony/webpack-encore/blob/master/test/config/validator.js since that's where we have the other "no entries found" checks.

Reply:
Hi @Lyrkan, thanks for suggestion. I moved the code to `test/config/validator.js` file. I hope the PR is good to merge now :)